templates/run.yaml.j2

```
@@ -45,7 +45,7 @@
     name: '{{ project }}-{{ suite.repo | default(repo) | replace ('.', '_') | replace ('/', '-') }}-{{ suite.container }}-params'
     repo: '{{ suite.repo | default(repo) }}'
     container: '{{ suite.container }}'
-    port:{% if suite.dport is defined %} {{ suite.dport }}
+    port: {% if suite.dport is defined %} {{ suite.dport }}
```

It may result in trailing white spaces. Would you wind removing them ? 

```suggestion
    port: {% if suite.dport is defined %}{{ suite.dport }}
```